src/argosim/antenna_utils.py

```diff
@@ -112,6 +112,42 @@ def random_antenna_arr(n_antenna=3, E_lim=1000, N_lim=1000, U_lim=0):
     # Return list of 'n' antenna locations (x_i, y_i) randomly distributed.
     return np.array([random_antenna_pos(E_lim, N_lim, U_lim) for i in range(n_antenna)])
 
+def uni_antenna_array(n_antenna_E=32, n_antenna_N=32, E_lim=800, N_lim=800, U_lim=0):
```

Comment:
I suggest adding default values as float for the limits, i.e., `E_lim=800., N_lim=800., U_lim=0.`